contrib/intel/jenkins/tests.py

```diff
@@ -211,6 +211,39 @@ def execute_cmd(self, shmem_testname):
         outputcmd = shlex.split(command)
         common.run_command(outputcmd)
 
+class ZeTest(Test):
```

Comment:
Update name to FabtestsZE to distinguish from other ZE testing to be added later